packages/edit-site/src/components/editor/index.js

```diff
@@ -349,6 +402,15 @@ export default function Editor( { isLoading, onClick } ) {
 							secondarySidebar: secondarySidebarLabel,
 						} }
 					/>
+					<Sidebar
+						onActionPerformed={ onActionPerformed }
+						extraPanels={
+							! isEditingPage && (
+								<PluginTemplateSettingPanel.Slot />
```

Comment:
The position of this slot has changed in the DOM. This slot is also deprecated, so I'm not sure we should make the effort to keep the previous position or not.